src/pages/SearchList/Calendar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useState } from "react";
```

Comment:
동일 패키지이기 때문에 한 줄로 import하실 수 있습니다.
```suggestion
import React, { useState } from "react";
```